db/migrate/20251008050307_delete_post_localizations_with_links.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+class DeletePostLocalizationsWithLinks < ActiveRecord::Migration[8.0]
+  def up
+    execute <<~SQL
+      DELETE FROM post_localizations
+      WHERE raw LIKE '%https://%'
+         OR cooked LIKE '%https://%'
```

Comment:
Should we also check for `http`?

Reply:
😆 

I would like to say no, but apparently we have `force_https` so I would have to. :sadpikachu: